modules/arrow/fixed-package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "version": "4.0.0",
+  "name": "apache-arrow",
+  "unpkg": "Arrow.es2015.min.js",
+  "jsdelivr": "Arrow.es2015.min.js",
+  "browser": "Arrow.dom",
+  "main": "Arrow.node",
+  "module": "Arrow.dom.mjs",
+  "sideEffects": false,
+  "esm": {
+    "mode": "all",
+    "sourceMap": true
+  },
+  "types": "Arrow.node.d.ts",
+  "license": "Apache-2.0",
+  "description": "Apache Arrow columnar in-memory format",
+  "author": "Apache Software Foundation",
+  "homepage": "https://github.com/apache/arrow/blob/master/js/README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache/arrow.git"
+  },
+  "bugs": {
+    "url": "https://issues.apache.org/jira/projects/ARROW"
+  },
+  "keywords": [
+    "apache",
+    "arrow"
+  ],
+  "dependencies": {
+    "@types/flatbuffers": "^1.10.0",
+    "@types/node": "^14.14.37",
+    "@types/text-encoding-utf-8": "^1.0.1",
+    "command-line-args": "5.1.1",
+    "command-line-usage": "6.1.1",
+    "flatbuffers": "1.12.0",
+    "json-bignum": "^0.0.3",
+    "pad-left": "^2.1.0",
+    "text-encoding-utf-8": "^1.0.2",
+    "tslib": "^2.2.0"
+  },
+  "bin": {
+    "arrow2csv": "bin/arrow2csv.js"
+  },
+  "browser": {
```

Comment:
@trxcllnt If I add this my bundling works, but I assume it overwrites your existing browser field.

Reply:
I believe we're publishing v6.0.2 to fix this packaging issue.

Reply:
https://github.com/apache/arrow/pull/11742